package.json

```diff
@@ -4,9 +4,16 @@
   "description": "## Shop with clothes schema",
```

Comment:
It's application or schema?) Too much for schema I think))

Reply:
Done